exporter/logzioexporter/README.md

```diff
@@ -42,10 +48,10 @@ receivers:
   prometheus:
     config:
       scrape_configs:
-      - job_name: 'ratelimiter'
-        scrape_interval: 15s
-        static_configs:
-        - targets: [ "0.0.0.0:8889" ]
+        - job_name: 'ratelimiter'
+          scrape_interval: 15s
+          static_configs:
+            - targets: [ "0.0.0.0:8889" ]
```

Comment:
Split to separate PR?

Reply:
@mx-psi Help me to understand why, This part was in previous versions of this exporter.

Reply:
You can check out the [file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/logzioexporter/README.md)

Reply:
Maintainers on this repo like to split things into different PRs for better git history and easier review, so I was suggesting splitting this. I am fine with keeping it here if you document this change on the PR description too :)

Reply:
@mx-psi I understand. Thanks a lot for your help. I checked this specific change and fixed the indentation.